### ISSUE TITLE
Developer Portal secondary navigation missing role

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#10213]
### Description:

1. have the role defined `role="navigation"`
2. Set area label to secondary navigation `aria-label="Secondary navigation"`
3. Links focus twice once for 
`<a class="jss26 jss25" href="/devportal/apis">` tag and 

second for the div
`<div class="MuiButtonBase-root MuiListItem-root jss30 MuiListItem-gutters MuiListItem-button" tabindex="0" role="button" aria-disabled="false">`
One link should focus only once

![image](https://user-images.githubusercontent.com/20179540/110575608-28cba400-8185-11eb-879d-27956f2786cc.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
